index.js

```diff
@@ -298,7 +298,7 @@ Color.prototype = {
    mix: function(mixinColor, weight) {
       var color1 = this;
       var color2 = mixinColor;
-      var weight = weight || 50;
+      var weight = weight !== undefined ? weight : 50;
```

Comment:
looked into travis https://travis-ci.org/harthur/color/builds/68992232

> We use an explicit 'undefined' test instead of relying on the
fals-y-ness of the 'weight' value as a weight of 0 is switched to 50
which isn't intuitive.

have no questions now =)